src/CXml/Model/Classification.php

```diff
@@ -6,7 +6,6 @@
 
 use JMS\Serializer\Annotation as Serializer;
 
-#[Serializer\AccessorOrder(order: 'custom', custom: ['value'])]
```

Comment:
why you deleted this serializer property?

Reply:
whats the need?